<ui version="4.0" >
 <class>TonemappingWindow</class>
 <widget class="QMainWindow" name="TonemappingWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1337</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Tone Mapping: </string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../icons.qrc" >
    <normaloff>:/new/prefix1/images/qtpfsgui.png</normaloff>:/new/prefix1/images/qtpfsgui.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1337</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="documentationAction" />
    <addaction name="actionWhat_s_This" />
   </widget>
   <widget class="QMenu" name="menuView" >
    <property name="title" >
     <string>&amp;View</string>
    </property>
    <addaction name="actionFit_to_Window" />
    <addaction name="separator" />
    <addaction name="actionAsThumbnails" />
    <addaction name="actionCascade" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionFix_Histogram" />
    <addaction name="actionSave" />
    <addaction name="actionSaveAll" />
    <addaction name="actionClose_All" />
    <addaction name="separator" />
    <addaction name="actionFinishTM" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuView" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>Toolbar</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionViewTMdock" />
   <addaction name="separator" />
   <addaction name="actionFix_Histogram" />
   <addaction name="actionSave" />
   <addaction name="actionSaveAll" />
   <addaction name="separator" />
   <addaction name="actionClose_All" />
   <addaction name="separator" />
   <addaction name="actionFinishTM" />
   <addaction name="actionWhat_s_This" />
  </widget>
  <action name="actionFinishTM" >
   <property name="icon" >
    <iconset resource="../icons.qrc" >
     <normaloff>:/new/prefix1/images/exit.png</normaloff>:/new/prefix1/images/exit.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Exit</string>
   </property>
   <property name="iconText" >
    <string>Exit</string>
   </property>
   <property name="toolTip" >
    <string>Close Tone Mapping Window</string>
   </property>
   <property name="statusTip" >
    <string>Close Tone Mapping Window</string>
   </property>
   <property name="whatsThis" >
    <string>Close Tone Mapping Window</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionViewTMdock" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons.qrc" >
     <normaloff>:/new/prefix1/images/qtpfsgui.png</normaloff>:/new/prefix1/images/qtpfsgui.png</iconset>
   </property>
   <property name="text" >
    <string>Tone Mapping Panel</string>
   </property>
   <property name="iconText" >
    <string>Tone Mapping Panel</string>
   </property>
   <property name="toolTip" >
    <string>Show/Hide the Tone mapping Panel</string>
   </property>
   <property name="statusTip" >
    <string>Show/Hide the Tone mapping Panel</string>
   </property>
   <property name="whatsThis" >
    <string>Show/Hide the Tone mapping Panel</string>
   </property>
  </action>
  <action name="actionFix_Histogram" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons.qrc" >
     <normaloff>:/new/prefix1/images/histogram.png</normaloff>:/new/prefix1/images/histogram.png</iconset>
   </property>
   <property name="text" >
    <string>Adjust &amp;Levels...</string>
   </property>
   <property name="iconText" >
    <string>Adjust Levels...</string>
   </property>
   <property name="toolTip" >
    <string>Tune the histogram for the current LDR</string>
   </property>
   <property name="statusTip" >
    <string>Tune the histogram for the current LDR</string>
   </property>
   <property name="whatsThis" >
    <string>Tune the histogram for the current LDR</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons.qrc" >
     <normaloff>:/new/prefix1/images/filesave.png</normaloff>:/new/prefix1/images/filesave.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Save as...</string>
   </property>
   <property name="iconText" >
    <string>Save As...</string>
   </property>
   <property name="toolTip" >
    <string>Save the currently selected LDR</string>
   </property>
   <property name="statusTip" >
    <string>Save the currently selected LDR</string>
   </property>
   <property name="whatsThis" >
    <string>Save the currently selected LDR</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose_All" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons.qrc" >
     <normaloff>:/new/prefix1/images/fileclose.png</normaloff>:/new/prefix1/images/fileclose.png</iconset>
   </property>
   <property name="text" >
    <string>C&amp;lose All</string>
   </property>
   <property name="iconText" >
    <string>Close All</string>
   </property>
   <property name="toolTip" >
    <string>Close all the LDRs in the workspace</string>
   </property>
   <property name="statusTip" >
    <string>Close all the LDRs in the workspace</string>
   </property>
   <property name="whatsThis" >
    <string>Close all the LDRs in the workspace</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionAsThumbnails" >
   <property name="text" >
    <string>As &amp;Thumbnails</string>
   </property>
   <property name="iconText" >
    <string>As Thumbnails</string>
   </property>
   <property name="toolTip" >
    <string>Images as Thumbnails</string>
   </property>
   <property name="statusTip" >
    <string>Layout all LDRs as thumbnails.</string>
   </property>
   <property name="whatsThis" >
    <string>Layout all LDRs as thumbnails.</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionCascade" >
   <property name="text" >
    <string>&amp;Cascade</string>
   </property>
   <property name="statusTip" >
    <string>Cascade all LDRs in the workspace</string>
   </property>
   <property name="whatsThis" >
    <string>Cascade all LDRs in the workspace</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionFit_to_Window" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons.qrc" >
     <normaloff>:/new/prefix1/images/viewmagfit.png</normaloff>:/new/prefix1/images/viewmagfit.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Fit to Window</string>
   </property>
   <property name="statusTip" >
    <string>Fit the current LDR to its containing window.</string>
   </property>
   <property name="whatsThis" >
    <string>Fit the current LDR to its containing window.</string>
   </property>
   <property name="shortcut" >
    <string>W</string>
   </property>
  </action>
  <action name="actionWhat_s_This" >
   <property name="icon" >
    <iconset resource="../icons.qrc" >
     <normaloff>:/new/prefix1/images/whatsthis.png</normaloff>:/new/prefix1/images/whatsthis.png</iconset>
   </property>
   <property name="text" >
    <string>What's This?</string>
   </property>
   <property name="toolTip" >
    <string>Get information about the graphical elements</string>
   </property>
   <property name="whatsThis" >
    <string>Use this button to get information about the graphical elements</string>
   </property>
   <property name="shortcut" >
    <string>Shift+F1</string>
   </property>
  </action>
  <action name="documentationAction" >
   <property name="icon" >
    <iconset resource="../icons.qrc" >
     <normaloff>:/new/prefix1/images/help.png</normaloff>:/new/prefix1/images/help.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Documentation...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionSaveAll" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons.qrc" >
     <normaloff>:/new/prefix1/images/filesaveall.png</normaloff>:/new/prefix1/images/filesaveall.png</iconset>
   </property>
   <property name="text" >
    <string>Save All</string>
   </property>
   <property name="iconText" >
    <string>Save All</string>
   </property>
   <property name="toolTip" >
    <string>Saves all LDRs</string>
   </property>
   <property name="statusTip" >
    <string>Saves all LDRs</string>
   </property>
   <property name="whatsThis" >
    <string>Saves all LDRs</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actionFinishTM</sender>
   <signal>triggered()</signal>
   <receiver>TonemappingWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>866</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
